duneanaobj/StandardRecord/SRTrack.h

```diff
@@ -22,6 +22,12 @@ namespace caf
 
       float Evis  = -999.;   ///< Visible energy in voxels corresponding to this track
 
+      // Track characteristics
+      float len_gcm3 = -999.; ///< Track length in g/cm3
+      float E = -999; ///< Track energy in MeV
+      float len_cm = -999; ///< Track length in centimeter (actual physical distance)
```

Comment:
just so it's clear to me, the reason you need this is because the tracks curve and so (end - start) isn't a good measure of length?

Reply:
There is that, and that the TMS has a region of thin and thick steel plates. So 40cm of track in thin steel plate region is not equivalent to 40cm of track in thick steel plate region